src/test/java/com/iexec/common/chain/CredentialsAbstractServiceTests.java

```diff
@@ -46,11 +47,43 @@ public CredentialsServiceStub(String walletPassword, String walletPath) throws E
             super(walletPassword, walletPath);
         }
 
+        public CredentialsServiceStub(Credentials credentials) throws Exception {
+            super(credentials);
+        }
+
         public CredentialsServiceStub() throws Exception {
             super();
         }
     }
 
+    // begin constructor (Credentials)
```

Comment:
What about `// region` comments to enable smart folding?

Reply:
Updated here https://github.com/iExecBlockchainComputing/iexec-common/pull/352/commits/d62f3848167c6cfe05a57aac131253dbad855d19, thanks :)